src/openforms/submissions/templates/report/submission_report.html

```diff
@@ -36,6 +36,7 @@ <h1 class="title">{{ report.form.name }}</h1>
                 <p>{% blocktrans with submission_date=report.submission.completed_on trimmed %}
                     Submitted on: {{ submission_date }}
                 {% endblocktrans %}</p>
+                <p>{% trans "Report created on:" %} {% now "DATETIME_FORMAT" %}</p>
```

Comment:
please apply the blocktrans pattern like above so that we can arrange the order of sentences in the translations.

You also want to let the normal rendering of django template engine apply for the timestamp so that its properly localized for the respective languages.

Reply:
I couldn't find how to make it work (`{% blocktrans with creation_date={% now ... %} %}` is invalid and no workarounds are mentioned in docs).

> You also want to let the normal rendering of django template engine apply for the timestamp so that its properly localized for the respective languages.

This is already picked up by `DATETIME_FORMAT`, which follows the current locale

Reply:
docs: https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#now

Use `{% now '...' as now_str %}` and then use `{{ now_str }}` in the blocktranslate.

> This is already picked up by DATETIME_FORMAT, which follows the current locale

I missed that you apparently *must* provide a format string to the tag. Kinda stupid.

Reply:
> docs: https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#now

:facepalm: 

(a bit weird to have `with` and `as` as two different options to do aliases)